contracts/seed/Seed.sol

```diff
@@ -28,22 +29,22 @@ contract Seed {
     address public admin;
     uint256 public softCap;
     uint256 public hardCap;
-    uint256 public seedAmountRequired;    // Amount of seed required for distribution
-    uint256 public feeAmountRequired;     // Amount of seed required for fee
-    uint256 public price;
+    uint256 public seedAmountRequired; // Amount of seed required for distribution
+    uint256 public feeAmountRequired;  // Amount of seed required for fee
+    uint256 public price;              // price of a SeedToken, expressed in fundingTokens, with precision of 10**18
     uint256 public startTime;
-    uint256 public endTime;               // set by project admin, this is the last resort endTime to be applied when
-                                          //     maximumReached has not been reached by then
+    uint256 public endTime;            // set by project admin, this is the last resort endTime to be applied when
+                                       //     maximumReached has not been reached by then
     bool    public permissionedSeed;
     uint32  public vestingDuration;
     uint32  public vestingCliff;
     IERC20  public seedToken;
     IERC20  public fundingToken;
-    uint8   public fee;
+    uint256 public fee;
 
     bytes   public metadata;           // IPFS Hash
 
-    uint256 constant internal PCT_BASE        = 10 ** 18;  // // 0% = 0; 1% = 10 ** 16; 100% = 10 ** 18
+    uint256 constant internal PRECISION = 10 ** 18;  // 0% = 0; 1% = 10 ** 16; 100% = 10 ** 18
```

Comment:
Hmm, honestly I don't agree that `PCT_BASE` was a bad name, but if they think so... @kinspotting any comments on the naming?
From the point of comments - they are a bit misleading... How is `PRECISION` connected with percentage? It is just a basis for tokens, the same way 1 ethereum = 10 ** 18 wei we have 1 token = 10 ** 18 parts of the token kinda...
So I think we need to remove percentage...

It is even a bit more strange, I've seen below that `PRECISION` was multiplied by 100, so it makes me think that `PRECISION` is a value for 1%...

Reply:
let's roll back to the original naming and not over-complicate the discussion

Reply:
Okay, I'll implement that.

Reply:
done